lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -361,6 +361,10 @@ export async function getUpdatedPackageFiles(
             const packageFileContents =
               updatedFileContents[packageFile] ||
               (await getFile(packageFile, config.baseBranch));
+            // workaround, see #27319
+            if (config.packageFiles?.[packageFile]?.lockFiles) {
```

Comment:
I get lint error:
```
Property 'lockFiles' does not exist on type 'PackageFile<Record<string, any>>[]'.ts(2339)
```
Even though this type should be okay.

https://github.com/renovatebot/renovate/blob/cf70ad0cc6c1585867124432915982043cd20177/lib/workers/types.ts#L125

Reply:
`packageFiles` should be renamed, because it has manager as a key. I was expecting a different interface looking at this name. I would suggest something explicit like `packageFilesByManager`